### ISSUE TITLE
Not building on the Arduino Portenta C33 

### ISSUE THREAD

**Issue by metanav** [bitbank2/JPEGDEC#68]
The latest release 1.3.1 doesn't work at the Arduino Portenta C33 (Renesas R7FA6M5BH2CBG Arm Cortex-M33). It displays error messages as reported below while compiling it. It works if it is downgraded to the release 1.2.10.
```
~/Library/Arduino15/packages/arduino/tools/arm-none-eabi-gcc/7-2017q4/lib/gcc/arm-none-eabi/7.2.1/include/arm_neon.h:6650:1: error: inlining failed in call to always_inline 'uint8x8_t vdup_n_u8(uint8_t)': target specific option mismatch
 vdup_n_u8 (uint8_t __a)
 ^~~~~~~~~
In file included from ~/Documents/Arduino/libraries/JPEGDEC/src/JPEGDEC.cpp:32:0:
~/Documents/Arduino/libraries/JPEGDEC/src/jpeg.inl:2840:13: note: called from here
        u88A = vdup_n_u8(0xff); // Alpha set to FF
        ~~~~~^~~~~~~~~~~~~~~~~
In file included from ~/Documents/Arduino/libraries/JPEGDEC/src/jpeg.inl:35:0,
                 from ~/Documents/Arduino/libraries/JPEGDEC/src/JPEGDEC.cpp:32:
~/Library/Arduino15/packages/arduino/tools/arm-none-eabi-gcc/7-2017q4/lib/gcc/arm-none-eabi/7.2.1/include/arm_neon.h:10266:1: error: inlining failed in call to always_inline 'int16x4_t vld1_s16(const int16_t*)': target specific option mismatch
```

**Comment by bitbank2** [comment]
Fixed with new release (1.4.0) which also adds ESP32-S3 SIMD. Are you working with multiple hardware boards for a project, or just testing JPEGDEC for the sake of testing?

**Comment by metanav** [comment]
Thanks @bitbank2, the latest release (1.4.0) works! I usually work on multiple hardwares for different projects. Currently I am working on porting the Pimoroni Impression 5.7" e-ink library to the Arduino Portenta C33.
